templates/taxi/car_detail.html

```diff
@@ -12,7 +12,20 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <div class="driver-heading">
+    <h1>Drivers</h1>
+    {% if user in car.drivers.all %}
+      <form action="{% url "taxi:delete-current-driver" pk=car.id %}" method="post">
+        {% csrf_token %}
```

Comment:
you use here backward relationship but did you fetched it from db? because otherwise here will be N+1 issue